types/index.d.ts

```diff
@@ -3,41 +3,41 @@ declare module WebdriverIO {
         /**
```

Comment:
Thanks for the Pr, the output is not 100% correct, it also depends on what the config was, please check https://github.com/wswebcreation/wdio-image-comparison-service#test-result-outputs, can you also add that

Reply:
Can you please check now: https://github.com/wswebcreation/wdio-image-comparison-service/pull/26/commits/f295c91a135a0d24d54c12d284c84a1ac6b7cf7a
I try to consider all the options.

Reply:
This looks good, did you verify if it works? If so I can merge it and release it.

Thanks for the work

Reply:
So, it looks good but works bad =)
It's always choosing `number`.
There are two options:
simple - use `any` for `check` functions,
complicated - we can divide `check(...): number` and  `check(...): {}` but values which function takes should differ functions. This complicates to do because of parameter `returnAllCompareData` in the config and we can know it just in compile time.

Reply:
I created a new PR with all the types, can you take a look at it? https://github.com/wswebcreation/wdio-image-comparison-service/pull/27

Tnx